### ISSUE TITLE
Don't depend on `mattwynne/changelog:latest`

### ISSUE THREAD

**Issue by aslakhellesoy** [cucumber/action-create-github-release#5]
When we depend on custom Docker images, it should be images that the whole Cucumber core team can maintain.

**Comment by mattwynne** [comment]
Yeah I totally agree. This was just a hack to get things going. I moved the action repo over to our org (https://github.com/cucumber-actions/changelog-action) but I haven't got around to re-publishing the docker image. It's a bit tricky as GitHub puts a bunch of rules in the way of you running workflows in a fork, so you basically have to do the `docker push` by hand.

**Comment by mpkorstanje** [comment]
We have a versioned image pushed here:  https://github.com/cucumber/changelog/releases/tag/0.10.0

**Comment by mpkorstanje** [comment]
Looks like the image we released doesn't work quite so well.
